bridges/opentracing/Gemfile

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+source 'https://rubygems.org'
+
+gem 'opentelemetry-api', '~> 0.0', path: '../../api'
+gem 'opentelemetry-sdk', '~> 0.0', path: '../../sdk'
```

Comment:
Do you need the version constraint here? We don't have it in the Jaeger exporter or HTTP example Gemfiles.